lib/capybara/spec/session/visit_spec.rb

```diff
@@ -17,6 +17,17 @@
     expect(@session).to have_content('Another World')
   end
 
+  # TODO: This test passes on racktest and selenium - although I think all it's really testing
+  # is that scheme defaults to http if not specified as anything else
```

Comment:
Yep - as I said in the other comment - if app_host is set it'll default to it's scheme, if not it'll default to http since thats set for the server url in #visit, and if not then it'll just end up defaulting to whatever the browser defaults to (probably file:)  --  I should probably just delete this test